meinberlin/assets/scss/components/_tile.scss

```diff
@@ -1,5 +1,48 @@
 .tile {
-    padding: $padding;
+    display: flex;
+    flex-direction: column;
     border: 1px solid $border-color;
     margin-bottom: $padding;
 }
+
+.tile__hint {
+    margin-bottom: 0;
+
+    color: $brand-danger;
+    font-size: $font-size-sm;
+}
+
+.tile__head {
+    position: relative;
+
+    &:before {
+        content: ' ';
+        width: 100%;
+        height: 6em;
+
+        position: absolute;
+        bottom: 0;
+
+        background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8));
+    }
+}
+
+.tile__badge {
+    height: 3.4em;
+    width: 3.4em;
+
+    position: absolute;
+    bottom: 0.8em;
+    right: 0.8em;
+}
+
+.tile__body {
+    flex-grow: 1;
```

Comment:
The [spec](https://www.w3.org/TR/css-flexbox-1/#flex-components) recommends to use the `flex` shorthand. I am not sure whether we should follow that advice.